cmd/builder/internal/command.go

```diff
@@ -59,7 +59,7 @@ func Command() (*cobra.Command, error) {
 	}
 
 	// the external config file
-	cmd.Flags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.otelcol-builder.yaml)")
+	cmd.Flags().StringVar(&cfgFile, "config", "$HOME/.ocb.yaml", "config file (default is $HOME/.ocb.yaml)")
```

Comment:
This is a breaking change right?  Is it ok to break this in 1 release or do we need to take a deprecated approach?  I'm thinking of situations where users of the ocb are already taking advantage of the default `.otelcol-builder.yaml` file and that would stop working when this is released.  Could break CICD pipelines.

Reply:
Yeh, it's a breaking change. We could revert (at the cost of some inconsistency), or we could fall back to `.otelcol-builder.yaml` if `.ocb.yaml` isn't present.

Medium term, I'm wondering whether it makes sense to take the config file only as an argument, not a flag, but I need to do some more research and learning beforehand so that I can make a good proposal :)

Reply:
Falling back to `.otelcol-builder.yaml` and adding a warning about it being deprecated in the future seems like a safe way to go.

Reply:
Since we need to add a fallback, what I propose is
* deprecate the required`--config` flag
* retain the current `--config` flag implementation for backward compatibility
* default to requiring a single argument that is the path to the config file to build

From my perspective, the default $HOME location for config isn't that useful, since the config ought to be under source control. Compulsory options are better expressed as arguments than flags.

I'll push an update later today.

Reply:
I am good with this approach, but it does deviate from the collector's ergonomics of passing in configuration via a --config flag.

Reply:
> does deviate from the collector's ergonomics of passing in configuration via a --config flag

Yep, IMHO distinction is that ocb is a compiler that takes a config input and generates a corresponding output, whereas the collector is a long-running service, where it's config is part of the system.